### ISSUE TITLE
Training image size

### ISSUE THREAD

**Issue by ghost** [hellochick/PSPNet-tensorflow#56]
I'd like to train on [Kitti dataset](http://www.cvlibs.net/datasets/kitti/eval_semantics.php), which is of size **1242 x 375.** Would this work with the current pool size?

**Update:** Tried to train with the default parameters from `train.py`, but after 2000 iterations the loss is ~2.
